### DISCUSSION TITLE
I want to know some details about Silero-VAD

### DISCUSSION THREAD

**Discussion by YLQY** [snakers4/silero-vad#105] [Q&A] [answered]
## ❓ Help

Hello, thank you for the VAD tool provided. Our company is preparing to use this tool as a long audio cutting tool, but I have not found the implementation details and principle of this VAD, so that we can better optimize the algorithm model

Hello, I want to know the principle of Silero-VAD. Can you provide some relevant documents, thank you

**Comment by snakers4** [comment, marked as answer]
Hi,

https://github.com/snakers4/silero-vad#how-vad-works
